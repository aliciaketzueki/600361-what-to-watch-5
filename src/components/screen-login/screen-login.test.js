import React from "react";
import renderer from "react-test-renderer";
import Login from "./screen-login";
import {Router as BrowserRouter} from "react-router-dom";
import {Provider} from "react-redux";
import browserHistory from "../../browser-history";
import {store} from "../../index";

describe(`Render Login`, () => {
  it(`Should Login render correctly`, () => {
    const tree = renderer
    .create(
        <Provider store={store}>
          <BrowserRouter history={browserHistory}>
            <Login />
          </BrowserRouter>
        </Provider>
    )
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
