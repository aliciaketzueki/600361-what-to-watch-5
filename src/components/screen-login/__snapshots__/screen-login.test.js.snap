// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Login Should Login render correctly 1`] = `
<div
  className="user-page"
>
  <Header
    film={null}
    header={
      Object {
        "headClass": "user-page__head",
        "title": "Sign in",
      }
    }
  />
  <div
    className="sign-in user-page__content"
  >
    <FormLogin
      checkValid={[Function]}
      commentValid={true}
      email=""
      emailValid={true}
      formErrors={
        Object {
          "email": "",
          "password": "",
          "textReview": "",
        }
      }
      formValid={false}
      handleFieldChange={[Function]}
      handleSubmit={[Function]}
      password=""
      passwordValid={true}
      rating="3"
      textReview=""
    />
  </div>
  <Footer />
</div>
`;
